lib/cAst.mli

```diff
@@ -0,0 +1,22 @@
+(************************************************************************)
+(*  v      *   The Coq Proof Assistant  /  The Coq Development Team     *)
+(* <O___,, *   INRIA - CNRS - LIX - LRI - PPS - Copyright 1999-2017     *)
+(*   \VV/  **************************************************************)
+(*    //   *      This file is distributed under the terms of the       *)
+(*         *       GNU Lesser General Public License Version 2.1        *)
+(************************************************************************)
+
+(** The ast type contains generic metadata for AST nodes *)
+type 'a ast = private {
```

Comment:
Well, there are these:
- [Unit.t](http://ocaml-batteries-team.github.io/batteries-included/hdoc2/BatUnit.html)
- [Bool.t](http://ocaml-batteries-team.github.io/batteries-included/hdoc2/BatBool.html)
- [Int.t](http://ocaml-batteries-team.github.io/batteries-included/hdoc2/BatInt.html)
- [Float.t](http://ocaml-batteries-team.github.io/batteries-included/hdoc2/BatFloat.html)
- [Int32.t](http://ocaml-batteries-team.github.io/batteries-included/hdoc2/BatInt32.html)
- [Int64.t](http://ocaml-batteries-team.github.io/batteries-included/hdoc2/BatInt64.html)

Although Coq source code is far from being that consistent.